src/entities/crm/crm.ts

```diff
@@ -337,6 +341,32 @@ export class CRM extends UniverseEntity<CRMPayload, CRMRawPayload> {
       throw this.handleError(new CRMAssociateUsersError(undefined, { error: err }))
     }
   }
+
+  public async createExternalUserFromPerson (payload: PersonPayload): Promise<number | undefined> {
+    if (this.id === null || this.id === undefined) throw new TypeError('CRM createExternalUserFromPerson requires id to be set.')
+
+    const data = {
+      person: payload.person
+    }
+
+    try {
+      const opts = {
+        method: 'PUT',
+        url: `${this.universe.universeBase}/${this.endpoint}/${this.id}/create_external_user_from_person`,
+        headers: {
+          'Content-Type': 'application/json; charset=utf-8',
+          'Content-Length': '0'
```

Comment:
Just remove this line, as content length is not 0, you pass an id.

Reply:
done